modules/ROOT/pages/legacy-search-experience.adoc

```diff
@@ -0,0 +1,323 @@
+= Legacy Search Experience
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+The legacy search experience is also known as the old search option on the Exchange UI. This search experience is available in Anypoint Platform Private Cloud Edition, MuleSoft Government Cloud, the US cloud, and the EU cloud. This search experience does not include enhanced options for searching by rank, tags, categories, organizations, and lifecycle states. 
```

Comment:
```suggestion
The legacy search experience is also known as the old search option on the Exchange UI. This search experience is available in Anypoint Platform Private Cloud Edition, MuleSoft Government Cloud, the US cloud, and the EU cloud. This search experience does not include enhanced options for searching by rank, organizations, and lifecycle states. 
```
Tags and categories can be used for filtering using the "query language" describe below.